src/CoreMidi/MidiStructs.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Runtime.InteropServices;
+
+using ObjCRuntime;
+using CoreFoundation;
+using Foundation;
+
+
+namespace CoreMidi {
+
+#if NET
+	[SupportedOSPlatform ("ios14.0")]
+	[SupportedOSPlatform ("maccatalyst")]
+	[SupportedOSPlatform ("tvos14.0")]
+	[SupportedOSPlatform ("macos11.0")]
+#else
+	[iOS (14,0), Mac (11,0), Watch (8,0), TV (14,0)]
+#endif
+	[StructLayout (LayoutKind.Sequential)]
+	public struct MidiEventList
+	{
+#if !COREBUILD
+		public MidiProtocolId protocol;
+#endif
```

Comment:
There are a couple of issues here:

1. Structs must have the same size between COREBUILD and !COREBUILD.
2. Public API must not start with a lower-cased letter.
3. It's often best to have private fields and public property accessors for structs, because it makes things easier if Apple changes their structs, and also if we need to adjust due to marshalling reasons (which is even more important for this struct, since it contains a variable-length array).

So something like this instead.

```cs
		int protocol;
		...

#if !COREBUILD
		public MidiProtocolId Protocol {
			get => (MidiProtocolId) protocol;
			set => protocol = (int) value;
		}
#endif
```